### ISSUE TITLE
The Apps debug binding command is nor working properly

### ISSUE THREAD

**Issue by DHaussermann** [mattermost/mattermost-plugin-apps#308]
The apps debug bindings. 
Is seems there is an issue for an expired session. Possibly related to recent session changes. 
![image](https://user-images.githubusercontent.com/35588783/159073606-b299aa9b-a092-4dcb-a3f5-72a4d8b4064d.png)

Steps:
- Install latest Apps plugin from `master` branch.
- Run `/apps debug bindings test`

**Comment by DHaussermann** [comment]
@hanzei I also see another issue where on a fresh install - the Test-App is creating the bot but there is no DM sent from the bot on a successful install. 
![image](https://user-images.githubusercontent.com/35588783/159178885-66561d04-73d1-4914-bee0-b1e5427e0d24.png)
0/5 - This may just be a symptom of this same issue rather than a separate problem with the test app. Please advise if I should create an issue in the test-app repo to further investigate this.

**Comment by hanzei** [comment]
@DHaussermann I can't reproduce the 	issue with `/apps debug bindings test `. Are there more specific steps that you can provide?


What kind of DM would you expect in the bot channel? Should there be a DM for every app? I want the confirm the expected behaviour.

**Comment by DHaussermann** [comment]
@hanzei I will try again now to repro. Maybe from a cleaner state. 

Regarding the DMs - I don't recall for Zendesk but, for Service Now and the Test-App when the install completed, the bot created by the install process would send the user a DM.

**Comment by DHaussermann** [comment]
@hanzei I re-tested on a fresh server and the the bindings debugs seems fine but I still don't get the bot DM.

![image](https://user-images.githubusercontent.com/35588783/160698664-1d431c3e-46df-48ac-84a1-e26628318056.png)

**Comment by hanzei** [comment]
@levb Is there supposed to be a DM?

**Comment by levb** [comment]
No, https://github.com/mattermost/mattermost-app-servicenow/blob/master/function/install.go

**Comment by hanzei** [comment]
@DHaussermann Can this issue be closed then?

**Comment by DHaussermann** [comment]
@hanzei I think so. One last thing on this...
The "missing" bot DMs may be completely unrelated to apps. 
I think it's possible the DMs I recall may simply be the generic "I'm a new bot add me to some teams" and had nothing to do with the apps themselves. 
 
@mickmister informed me that the way bots are created has changed and so there may be valid reason why I no longer see the DMs 

Is this consistent with your understanding?

**Comment by hanzei** [comment]
@DHaussermann Do you have an example message from an "older app install" at hand?

**Comment by hanzei** [comment]
@DHaussermann You had a great sense on what is going on here. In the server, as message is send to the creator of the bot that informs them that the bot needs to be added to channel/teams. https://github.com/mattermost/mattermost-server/blob/f2c955b3d3bdec2aa443811b005ec8c0c317c0b7/app/bot.go#L62-L90

`Please add me to teams and channels you want me to interact in. To do this, use the browser or Mattermost Desktop App.`

Given that we don't set an `OwnerId` any longer (https://github.com/mattermost/mattermost-plugin-apps/blob/c6066f18a9216d2646598d18fa7d9a46bf4b40a0/server/proxy/install.go#L195-L199) this message is not longer send.

I'm queuing this ticket up for triage to decide if that message is helpful or not.

**Comment by mickmister** [comment]
@DHaussermann Should this be closed, as I think we agreed that it is the individual App's responsobility to create this DM upon installation?
